charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -21,7 +21,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 46.6.0
+version: 46.7.0
 appVersion: v0.65.1
```

Comment:
Can you also bump the prometheus operator to 0.65.2?

Reply:
Done